src/pages/studyView/charts/scatterPlot/StudyViewDensityScatterPlot.tsx

```diff
@@ -103,11 +133,33 @@ export default class StudyViewDensityScatterPlot
     }
 
     @computed get plotDomain() {
-        // enforce plot constraints - because of dot size and wanting them to be
-        //  right up against each other, we cant have less than yBinsMin on the y axis
+        const x = [
```

Comment:
maybe bring back comment.  like, what is plotDomain and why would it be initially undefined here?